sim/druid/insect_swarm.go

```diff
@@ -66,7 +70,7 @@ func (druid *Druid) registerInsectSwarmSpell() {
 		TickLength:    time.Second * 2,
 
 		OnSnapshot: func(sim *core.Simulation, target *core.Unit, dot *core.Dot, isRollover bool) {
-			dot.SnapshotBaseDamage = 215 + 0.2*dot.Spell.SpellPower()
+			dot.SnapshotBaseDamage = (215 + idolSpellPower) + 0.2*dot.Spell.SpellPower()
```

Comment:
do you need to add the idolSpellPower to the dot.Spell.SpellPower() so its multiplied by 0.2? or is it added to the base damage?